packages/orbit-components/src/Toast/ToastMessage.tsx

```diff
@@ -56,9 +56,10 @@ const ToastMessage = ({
       }
     >
       <div
+        /* eslint-disable-next-line tailwindcss/no-custom-classname */
         className={cx(
           "rounded-large bg-ink-dark p-xs relative w-full overflow-hidden will-change-transform",
-          "lm:max-w-[360px] lm:w-auto lm:p-sm [&_svg]:min-h-[20px]",
+          "lm:max-w-modal-extra-small lm:w-auto lm:p-sm [&_svg]:min-h-5",
```

Comment:
We should not use component based tokens outside of the scope of it component. In this case, it's a Modal class used on Toast. I know they have the same value, so we should probably discuss unifying so we could reuse it. But, for now, this should be kept as a custom value. Nice catch, though.